app/Repositories/Summit/DoctrineSummitEventRepository.php

```diff
@@ -616,12 +697,61 @@ protected function getOrderMappings()
 END
 SQL,
             'selection_plan' => 'selp.name',
-            'actions' => 'HIDDEN_COMPLETED_ACTIONS_COUNT'
+            'actions' => 'HIDDEN_COMPLETED_ACTIONS_COUNT',
             /*
             'event_type_capacity' => <<<SQL
 SQL,
             'speakers' => <<<SQL
 SQL,*/
+            'review_status' => <<<SQL
+    CASE
+    WHEN p is null THEN ''
+	WHEN selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted') THEN 'NoSubmitted'
+	WHEN (p.status = 'Received' OR p.status = 'Accepted') AND
+		selp IS NOT null AND (
+		(selp.submission_lock_down_presentation_status_date IS NOT NULL AND selp.submission_lock_down_presentation_status_date > '$now_utc') OR 
+		(
+			(selp.submission_begin_date > '$now_utc' OR selp.submission_end_date < '$now_utc') 
+			AND 
+			(selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+		)
+	) THEN 'InReview'
+	WHEN (p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
+		selp IS NOT null AND ( 
+		(selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+		(
+			(selp.submission_begin_date <= '$now_utc' AND selp.submission_end_date >= '$now_utc') 
+			OR 
+			(selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc')
+		)
+	) THEN 'Published'
+    WHEN (p.status = 'Received' OR p.status = 'Accepted') AND e.published = 0 AND 
+        (selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc') 
+        AND EXISTS (
+            SELECT ___sp331.id
+            FROM models\summit\SummitSelectedPresentation ___sp331 
+            JOIN ___sp331.presentation ___p331 
+            JOIN ___p331.category ___pc331                
+            JOIN ___sp331.list ___spl331 WITH ___spl331.list_type = 'Group' AND ___spl331.list_class = 'Session' 
+            WHERE ___p331.id = e.id 
+            AND ___sp331.collection = 'selected'
+            AND ___sp331.order > ___pc331.session_count                  
+    ) THEN 'Accepted'
+    WHEN (p.status = 'Received' OR p.status = 'Accepted') AND e.published = 0 AND 
+        (selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc') 
```

Comment:
u need to check here the proper conditions for
NOT lockDownPeriod AND ( submissionOpen OR selectionClosed )